src/discljord/events/middleware.clj

```diff
@@ -87,7 +87,7 @@
     (fn [handler]
       (fn [event-type event-data & more]
         (when-not @reduced
-          (let [res (reducer handler (into [event-type event-data] more))]
+          (let [res (reducer handler (vec (concat [event-type event-data] more)))]
```

Comment:
Doesn't `into` do the same as `vec` and `concat`?

Reply:
It doesn't do the same, but it is semantically equivalent, yes. I reverted this because this change is also not part of this PR.